src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { ActivatedRoute } from '@angular/router';
```

Comment:
Even when the component doesn't do anything at one point, we still need to do the tests and do at least one expect, ALWAYS.